schema/cassandra/temporal/versioned/v1.0/schema.cql

```diff
@@ -1,7 +1,7 @@
 CREATE TYPE serialized_event_batch (
   encoding_type text,
   version       int,
-  data          blob,
+  data          blob
```

Comment:
Could you update https://github.com/temporalio/temporal/blob/87a48bb9c1a4e4921b74bc33f114f12334a29037/schema/cassandra/temporal/schema.cql#L4 as well

Reply:
Thanks for the hint. By the way in case there is a full schema maintained, it would be better to have the newest schema packaged somehow...